### ISSUE TITLE
Using Frank with plain Rspec..

### ISSUE THREAD

**Issue by 34code** [moredip/Frank#139]
Is this possible? Or do I _have_ to use cucumber to write my tests... I personally prefer writing tests in Rspec and was wondering if I can use Rspec to test my iOS apps with Frank..

If it is possible, could someone give me rough code to start an rspec project using frank?

If its not possible, could someone suggest how to make it possible? I would be happy to fork and start working on rspec support...

**Comment by moredip** [comment]
Yes, this is possible. I don't have example code to hand - the project I did this on isn't open source unfortunately. I should write a blog post or add a page to the testingwithfrank.com site with details. I'll leave this issue open as a reminder to do so. 

I would also suggest asking on the Frank mailing list. Someone else on the list has probably already done this.
